mas-foundation/src/main/java/com/ca/mas/core/policy/PolicyManager.java

```diff
@@ -113,7 +158,7 @@ public MASResponse execute(RequestInfo requestInfo, Route<MASResponse> function)
 
 
     public void close() {
-        close(defaultPolicy);
```

Comment:
The defaultPolicy contains all assertion, close the defaultPolicy should be good.

Reply:
done!